internal/install/install.go

```diff
@@ -99,16 +110,8 @@ func writeStackResources(elasticPackagePath *locations.LocationManager) error {
 		return errors.Wrapf(err, "creating directory failed (path: %s)", elasticPackagePath.PackagesDir())
 	}
 
-	err = writeStaticResource(err, filepath.Join(elasticPackagePath.StackDir(), "kibana.config.yml"), kibanaConfigYml)
-	err = writeStaticResource(err, filepath.Join(elasticPackagePath.StackDir(), "healthcheck.sh"), kibanaHealthcheckSh)
-	err = writeStaticResource(err, filepath.Join(elasticPackagePath.StackDir(), "snapshot.yml"), snapshotYml)
-	err = writeStaticResource(err, filepath.Join(elasticPackagePath.StackDir(), "package-registry.config.yml"), packageRegistryConfigYml)
-	err = writeStaticResource(err, filepath.Join(elasticPackagePath.StackDir(), "Dockerfile.package-registry"), packageRegistryDockerfile)
+	return profile.CreateProfile(elasticPackagePath.StackDir(), profile.DefaultProfile, false)
```

Comment:
Since we're no longer keeping profiles specific to `elastic-package stack` commands, does it make sense now to store profile files under something other than the stack dir? Maybe a new dir that's a sibling of the stack dir, e.g. `profiles` or something?

Reply:
We could. My thinking was that the less we moved stuff around, the less weird paths we'll need to send to docker, since we already have stuff like `../../tmp/service_logs/`, which I'm not a fan of as-is.